src/seedu/addressbook/data/person/Person.java

```diff
@@ -87,4 +87,13 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    @Override
+    public String getPrintableString(Printable... printables) {
```

Comment:
- need to use this method somewhere
- all non-trivial methods need header comments